internal/goofys.go

```diff
@@ -1056,11 +1056,7 @@ func (fs *Goofys) MkDir(
 	}
 
 	parent.mu.Lock()
-
-	fs.mu.Lock()
```

Comment:
We don't need this lock since this lock is already acquired in the beginning of the method.